test/integration/admin_tenants_test.rb

```diff
@@ -0,0 +1,123 @@
+require "test_helper"
+
+class AdminUserTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+
+  test "an admin sees a tenants link on the dashboard" do
+    admin = create(:admin)
+    ApplicationController.any_instance.stubs(:current_user).returns(admin)
+
+    visit admin_dashboard_path
+
+    assert page.has_content?("Tenants")
```

Comment:
Actually, maybe we don't even want this. There is currently a link in the navbar of the Admin page that says "All borrowers" which leads to the same place. I guess we need to discuss exactly what we want on the Admin dashboard, and which links should be in the navbar vs the body.